<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prutype</class>
 <widget class="QDialog" name="prutype">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>634</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget
{
border-color: #2f3640;
border-style: solid;
border-width: 1px;
}
QCheckBox
{
border-width: 0px;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>信息查询</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="PersonInfo">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>个人信息</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="YearInfo">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>年度履历</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="punishInfo">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>工作日历</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="dutycbx">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>组织架构</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>奖惩管理</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="reward">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>奖励申请</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="report">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>检举</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="audit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>奖惩审核</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>用户管理</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="staff">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>账号管理</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="manageRule">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>管理条例</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="purType_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>岗位管理</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addtype">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
       </property>
       <property name="text">
        <string>新增类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deltype">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
       </property>
       <property name="text">
        <string>删除类型</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ptsave">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
       </property>
       <property name="text">
        <string>保存权限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ptcancel">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 6px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="prutype_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
